2018-07-31 22:22:14.203 EDT [54000] FATAL:  lock file "postmaster.pid" already exists
2018-07-31 22:22:14.203 EDT [54000] HINT:  Is another postmaster (PID 51543) running in data directory "/usr/local/var/postgres"?
2018-07-31 22:22:26.489 EDT [54018] LOG:  listening on IPv6 address "::1", port 5432
2018-07-31 22:22:26.489 EDT [54018] LOG:  listening on IPv6 address "fe80::1%lo0", port 5432
2018-07-31 22:22:26.489 EDT [54018] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2018-07-31 22:22:26.490 EDT [54018] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2018-07-31 22:22:26.519 EDT [54019] LOG:  database system was shut down at 2018-07-31 22:22:23 EDT
2018-07-31 22:22:26.524 EDT [54018] LOG:  database system is ready to accept connections
2018-08-03 11:33:17.635 EDT [29886] ERROR:  duplicate key value violates unique constraint "crime_state_abbrev_key"
2018-08-03 11:33:17.635 EDT [29886] DETAIL:  Key (abbrev)=(abbreviation) already exists.
2018-08-03 11:33:17.635 EDT [29886] STATEMENT:  INSERT INTO "crime_state" ("name", "abbrev", "fips", "assoc_press", "census_division_name", "standard_federal_region", "census_region_name", "census_region", "circuit_court", "census_division") VALUES ('name', 'abbreviation', 'fips_state', 'assoc_press', 'census_division_name', 'standard_federal_region', 'census_region_name', 'census_region', 'circuit_court', 'census_division') RETURNING "crime_state"."id"
2018-08-03 11:33:17.636 EDT [29886] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 11:33:17.636 EDT [29886] STATEMENT:  SELECT 1
2018-08-03 11:33:45.138 EDT [29989] ERROR:  duplicate key value violates unique constraint "crime_state_abbrev_key"
2018-08-03 11:33:45.138 EDT [29989] DETAIL:  Key (abbrev)=(abbreviation) already exists.
2018-08-03 11:33:45.138 EDT [29989] STATEMENT:  INSERT INTO "crime_state" ("name", "abbrev", "fips", "assoc_press", "census_division_name", "standard_federal_region", "census_region_name", "census_region", "circuit_court", "census_division") VALUES ('name', 'abbreviation', 'fips_state', 'assoc_press', 'census_division_name', 'standard_federal_region', 'census_region_name', 'census_region', 'circuit_court', 'census_division') RETURNING "crime_state"."id"
2018-08-03 11:33:45.139 EDT [29989] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 11:33:45.139 EDT [29989] STATEMENT:  SELECT 1
2018-08-03 11:40:04.063 EDT [30530] ERROR:  duplicate key value violates unique constraint "crime_state_abbrev_key"
2018-08-03 11:40:04.063 EDT [30530] DETAIL:  Key (abbrev)=("AL") already exists.
2018-08-03 11:40:04.063 EDT [30530] STATEMENT:  INSERT INTO "crime_state" ("name", "abbrev", "fips", "assoc_press", "census_division_name", "standard_federal_region", "census_region_name", "census_region", "circuit_court", "census_division") VALUES ('"Alabama"', '"AL"', '"1"', '"Ala."', '"East South Central"', '"IV"', '"South"', '"3"', '"11"', '"6"') RETURNING "crime_state"."id"
2018-08-03 11:40:04.063 EDT [30530] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 11:40:04.063 EDT [30530] STATEMENT:  SELECT 1
2018-08-03 11:44:27.297 EDT [31186] ERROR:  duplicate key value violates unique constraint "crime_state_abbrev_key"
2018-08-03 11:44:27.297 EDT [31186] DETAIL:  Key (abbrev)=("AL") already exists.
2018-08-03 11:44:27.297 EDT [31186] STATEMENT:  INSERT INTO "crime_state" ("name", "abbrev", "fips", "assoc_press", "census_division_name", "standard_federal_region", "census_region_name", "census_region", "circuit_court", "census_division") VALUES ('"Alabama"', '"AL"', '"1"', '"Ala."', '"East South Central"', '"IV"', '"South"', '"3"', '"11"', '"6"') RETURNING "crime_state"."id"
2018-08-03 11:44:27.297 EDT [31186] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 11:44:27.297 EDT [31186] STATEMENT:  SELECT 1
2018-08-03 11:47:07.145 EDT [31589] ERROR:  duplicate key value violates unique constraint "crime_state_abbrev_key"
2018-08-03 11:47:07.145 EDT [31589] DETAIL:  Key (abbrev)=("AL") already exists.
2018-08-03 11:47:07.145 EDT [31589] STATEMENT:  INSERT INTO "crime_state" ("name", "abbrev", "fips", "assoc_press", "census_division_name", "standard_federal_region", "census_region_name", "census_region", "circuit_court", "census_division") VALUES ('"Alabama"', '"AL"', '"1"', '"Ala."', '"East South Central"', '"IV"', '"South"', '"3"', '"11"', '"6"') RETURNING "crime_state"."id"
2018-08-03 11:47:07.145 EDT [31589] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 11:47:07.145 EDT [31589] STATEMENT:  SELECT 1
2018-08-03 14:46:59.947 EDT [40063] ERROR:  column "ori7" of relation "crime_agency" does not exist at character 159
2018-08-03 14:46:59.947 EDT [40063] STATEMENT:  INSERT INTO "crime_agency" ("name", "street_number", "street_name", "room", "city_id", "state_name_id", "zip_code", "website", "agency_notes", "agency_type", "ori7", "ori9") VALUES ('testing', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '1', NULL, NULL) RETURNING "crime_agency"."id"
2018-08-03 14:46:59.948 EDT [40063] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 14:46:59.948 EDT [40063] STATEMENT:  SELECT 1
2018-08-03 14:47:47.084 EDT [40063] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 14:47:47.084 EDT [40063] STATEMENT:  SELECT "crime_state"."id", "crime_state"."name", "crime_state"."abbrev", "crime_state"."fips", "crime_state"."assoc_press", "crime_state"."census_division_name", "crime_state"."standard_federal_region", "crime_state"."census_region_name", "crime_state"."census_region", "crime_state"."circuit_court", "crime_state"."census_division" FROM "crime_state" WHERE "crime_state"."abbrev" = 'NJ'
2018-08-03 14:48:37.500 EDT [40063] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 14:48:37.500 EDT [40063] STATEMENT:  SELECT "crime_state"."id", "crime_state"."name", "crime_state"."abbrev", "crime_state"."fips", "crime_state"."assoc_press", "crime_state"."census_division_name", "crime_state"."standard_federal_region", "crime_state"."census_region_name", "crime_state"."census_region", "crime_state"."circuit_court", "crime_state"."census_division" FROM "crime_state" WHERE "crime_state"."abbrev" = 'NJ'
2018-08-03 14:48:49.695 EDT [40063] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 14:48:49.695 EDT [40063] STATEMENT:  SELECT "crime_state"."id", "crime_state"."name", "crime_state"."abbrev", "crime_state"."fips", "crime_state"."assoc_press", "crime_state"."census_division_name", "crime_state"."standard_federal_region", "crime_state"."census_region_name", "crime_state"."census_region", "crime_state"."circuit_court", "crime_state"."census_division" FROM "crime_state" WHERE "crime_state"."abbrev" = 'NJ'
2018-08-03 14:48:59.301 EDT [40063] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 14:48:59.301 EDT [40063] STATEMENT:  SELECT "crime_state"."id", "crime_state"."name", "crime_state"."abbrev", "crime_state"."fips", "crime_state"."assoc_press", "crime_state"."census_division_name", "crime_state"."standard_federal_region", "crime_state"."census_region_name", "crime_state"."census_region", "crime_state"."circuit_court", "crime_state"."census_division" FROM "crime_state" WHERE "crime_state"."abbrev" = 'NJ'
2018-08-03 14:49:06.859 EDT [40063] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 14:49:06.859 EDT [40063] STATEMENT:  SELECT "crime_state"."id", "crime_state"."name", "crime_state"."abbrev", "crime_state"."fips", "crime_state"."assoc_press", "crime_state"."census_division_name", "crime_state"."standard_federal_region", "crime_state"."census_region_name", "crime_state"."census_region", "crime_state"."circuit_court", "crime_state"."census_division" FROM "crime_state" WHERE "crime_state"."abbrev" = 'NJ' LIMIT 21
2018-08-03 14:49:16.018 EDT [40063] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 14:49:16.018 EDT [40063] STATEMENT:  SELECT "crime_state"."id", "crime_state"."name", "crime_state"."abbrev", "crime_state"."fips", "crime_state"."assoc_press", "crime_state"."census_division_name", "crime_state"."standard_federal_region", "crime_state"."census_region_name", "crime_state"."census_region", "crime_state"."circuit_court", "crime_state"."census_division" FROM "crime_state" LIMIT 21
2018-08-03 14:50:13.248 EDT [40383] ERROR:  column "ori7" of relation "crime_agency" does not exist at character 159
2018-08-03 14:50:13.248 EDT [40383] STATEMENT:  INSERT INTO "crime_agency" ("name", "street_number", "street_name", "room", "city_id", "state_name_id", "zip_code", "website", "agency_notes", "agency_type", "ori7", "ori9") VALUES ('NJSP3', NULL, NULL, NULL, NULL, 30, NULL, NULL, NULL, '1', NULL, NULL) RETURNING "crime_agency"."id"
2018-08-03 14:50:13.248 EDT [40383] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 14:50:13.248 EDT [40383] STATEMENT:  SELECT 1
2018-08-03 14:51:30.426 EDT [40534] ERROR:  column "ori7" of relation "crime_agency" does not exist at character 159
2018-08-03 14:51:30.426 EDT [40534] STATEMENT:  INSERT INTO "crime_agency" ("name", "street_number", "street_name", "room", "city_id", "state_name_id", "zip_code", "website", "agency_notes", "agency_type", "ori7", "ori9") VALUES ('njsp3', NULL, NULL, NULL, NULL, 30, NULL, NULL, NULL, '1', NULL, NULL) RETURNING "crime_agency"."id"
2018-08-03 14:51:30.426 EDT [40534] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-03 14:51:30.426 EDT [40534] STATEMENT:  SELECT 1
2018-08-05 08:26:36.519 EDT [57372] ERROR:  duplicate key value violates unique constraint "crime_state_abbrev_key"
2018-08-05 08:26:36.519 EDT [57372] DETAIL:  Key (abbrev)=(AL) already exists.
2018-08-05 08:26:36.519 EDT [57372] STATEMENT:  INSERT INTO "crime_state" ("name", "abbrev", "fips", "assoc_press", "census_division_name", "standard_federal_region", "census_region_name", "census_region", "circuit_court", "census_division") VALUES ('Alabama', 'AL', '1', 'Ala.', 'East South Central', 'IV', 'South', '3', '11', '6') RETURNING "crime_state"."id"
2018-08-05 08:26:36.519 EDT [57372] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-05 08:26:36.519 EDT [57372] STATEMENT:  SELECT 1
2018-08-07 16:59:14.078 EDT [87617] ERROR:  duplicate key value violates unique constraint "crime_release_hj_url_key"
2018-08-07 16:59:14.078 EDT [87617] DETAIL:  Key (hj_url)=(/njsp-crime-reports/pdfs/20171025_crimetrend.pdf) already exists.
2018-08-07 16:59:14.078 EDT [87617] STATEMENT:  INSERT INTO "crime_release" ("file_name", "file_type", "hj_url", "length", "date_released", "date_collected", "year_of_data", "time_period_covered", "data_source_id") VALUES ('20171025_crimetrend.pdf', '1', '/njsp-crime-reports/pdfs/20171025_crimetrend.pdf', 500, '2017-10-25'::date, NULL, 2017, '3', 1) RETURNING "crime_release"."id"
2018-08-07 16:59:14.078 EDT [87617] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-07 16:59:14.078 EDT [87617] STATEMENT:  SELECT 1
2018-08-10 16:45:30.123 EDT [48810] ERROR:  duplicate key value violates unique constraint "crime_release_hj_url_key"
2018-08-10 16:45:30.123 EDT [48810] DETAIL:  Key (hj_url)=(pdfs/20180810_crimetrend_2018.pdf) already exists.
2018-08-10 16:45:30.123 EDT [48810] STATEMENT:  INSERT INTO "crime_release" ("file_name", "file_type", "hj_url", "length", "date_released", "date_collected", "year_of_data", "frequency_type", "data_source_id") VALUES ('20180810_crimetrend_2018.pdf', '1', 'pdfs/20180810_crimetrend_2018.pdf', 568, '2018-08-10'::date, '2018-08-10'::date, 2018, '1', 1) RETURNING "crime_release"."id"
2018-08-10 16:45:30.124 EDT [48810] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-10 16:45:30.124 EDT [48810] STATEMENT:  SELECT 1
2018-08-10 16:48:24.227 EDT [48810] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-10 16:48:24.227 EDT [48810] STATEMENT:  SELECT COUNT(*) AS "__count" FROM "crime_release" WHERE "crime_release"."file_name" = '20180810_crimetrend_2018.pdf'
2018-08-14 17:26:54.570 EDT [23985] ERROR:  relation "django_celery_beat_solarschedule" does not exist
2018-08-14 17:26:54.570 EDT [23985] STATEMENT:  SELECT setval(pg_get_serial_sequence('"django_celery_beat_solarschedule"','id'), 1, false);
2018-08-14 17:27:29.721 EDT [24079] ERROR:  duplicate key value violates unique constraint "django_content_type_pkey"
2018-08-14 17:27:29.721 EDT [24079] DETAIL:  Key (id)=(1) already exists.
2018-08-14 17:27:29.721 EDT [24079] STATEMENT:  INSERT INTO "django_content_type" ("app_label", "model") VALUES ('django_celery_beat', 'solarschedule') RETURNING "django_content_type"."id"
2018-08-14 17:28:07.064 EDT [24229] ERROR:  duplicate key value violates unique constraint "django_content_type_pkey"
2018-08-14 17:28:07.064 EDT [24229] DETAIL:  Key (id)=(2) already exists.
2018-08-14 17:28:07.064 EDT [24229] STATEMENT:  INSERT INTO "django_content_type" ("app_label", "model") VALUES ('django_celery_beat', 'solarschedule') RETURNING "django_content_type"."id"
2018-08-14 17:28:43.668 EDT [24329] ERROR:  duplicate key value violates unique constraint "django_content_type_pkey"
2018-08-14 17:28:43.668 EDT [24329] DETAIL:  Key (id)=(3) already exists.
2018-08-14 17:28:43.668 EDT [24329] STATEMENT:  INSERT INTO "django_content_type" ("app_label", "model") VALUES ('django_celery_beat', 'solarschedule') RETURNING "django_content_type"."id"
2018-08-14 17:29:11.105 EDT [16885] ERROR:  update or delete on table "django_content_type" violates foreign key constraint "auth_content_type_id_508cf46651277a81_fk_django_content_type_id" on table "auth_permission"
2018-08-14 17:29:11.105 EDT [16885] DETAIL:  Key (id)=(38) is still referenced from table "auth_permission".
2018-08-14 17:29:11.105 EDT [16885] STATEMENT:  delete from "public"."django_content_type" where "id"='38'
2018-08-14 17:32:08.210 EDT [16873] ERROR:  cannot drop table auth_group because other objects depend on it
2018-08-14 17:32:08.210 EDT [16873] DETAIL:  constraint auth_group_permissio_group_id_689710a9a73b7457_fk_auth_group_id on table auth_group_permissions depends on table auth_group
	constraint auth_user_groups_group_id_33ac548dcf5f8e37_fk_auth_group_id on table auth_user_groups depends on table auth_group
2018-08-14 17:32:08.210 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:08.210 EDT [16873] STATEMENT:  drop table "public"."auth_group"
2018-08-14 17:32:09.140 EDT [16873] ERROR:  cannot drop table auth_permission because other objects depend on it
2018-08-14 17:32:09.140 EDT [16873] DETAIL:  constraint auth_user__permission_id_384b62483d7071f0_fk_auth_permission_id on table auth_user_user_permissions depends on table auth_permission
2018-08-14 17:32:09.140 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:09.140 EDT [16873] STATEMENT:  drop table "public"."auth_permission"
2018-08-14 17:32:09.602 EDT [16873] ERROR:  cannot drop table auth_user because other objects depend on it
2018-08-14 17:32:09.602 EDT [16873] DETAIL:  constraint auth_user_groups_user_id_4b5ed4ffdb8fd9b0_fk_auth_user_id on table auth_user_groups depends on table auth_user
	constraint auth_user_user_permiss_user_id_7f0938558328534a_fk_auth_user_id on table auth_user_user_permissions depends on table auth_user
	constraint django_admin_log_user_id_52fdd58701c5f563_fk_auth_user_id on table django_admin_log depends on table auth_user
	constraint executions_knelluser_user_id_7948349a51d96be2_fk_auth_user_id on table executions_knelluser depends on table auth_user
	constraint tastypie_apikey_user_id_ffeb4840e0b406b_fk_auth_user_id on table tastypie_apikey depends on table auth_user
2018-08-14 17:32:09.602 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:09.602 EDT [16873] STATEMENT:  drop table "public"."auth_user"
2018-08-14 17:32:10.026 EDT [16873] ERROR:  cannot drop table django_content_type because other objects depend on it
2018-08-14 17:32:10.026 EDT [16873] DETAIL:  constraint auth_content_type_id_508cf46651277a81_fk_django_content_type_id on table auth_permission depends on table django_content_type
2018-08-14 17:32:10.026 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:10.026 EDT [16873] STATEMENT:  drop table "public"."django_content_type"
2018-08-14 17:32:10.433 EDT [16873] ERROR:  cannot drop table djcelery_crontabschedule because other objects depend on it
2018-08-14 17:32:10.433 EDT [16873] DETAIL:  constraint djce_crontab_id_1d8228f5b44b680a_fk_djcelery_crontabschedule_id on table djcelery_periodictask depends on table djcelery_crontabschedule
2018-08-14 17:32:10.433 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:10.433 EDT [16873] STATEMENT:  drop table "public"."djcelery_crontabschedule"
2018-08-14 17:32:10.816 EDT [16873] ERROR:  cannot drop table djcelery_intervalschedule because other objects depend on it
2018-08-14 17:32:10.816 EDT [16873] DETAIL:  constraint dj_interval_id_20cfc1cad060dfad_fk_djcelery_intervalschedule_id on table djcelery_periodictask depends on table djcelery_intervalschedule
2018-08-14 17:32:10.816 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:10.816 EDT [16873] STATEMENT:  drop table "public"."djcelery_intervalschedule"
2018-08-14 17:32:11.238 EDT [16873] ERROR:  cannot drop table executions_agency because other objects depend on it
2018-08-14 17:32:11.238 EDT [16873] DETAIL:  constraint executions_f_agency_id_1ffba96eeb1c76fb_fk_executions_agency_id on table executions_facility depends on table executions_agency
	constraint executions_fl_agency_id_e060ff4833e9938_fk_executions_agency_id on table executions_flack depends on table executions_agency
2018-08-14 17:32:11.238 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:11.238 EDT [16873] STATEMENT:  drop table "public"."executions_agency"
2018-08-14 17:32:11.619 EDT [16873] ERROR:  cannot drop table executions_aggro because other objects depend on it
2018-08-14 17:32:11.619 EDT [16873] DETAIL:  constraint executions_cas_aggro_id_163fb51227db7977_fk_executions_aggro_id on table executions_case_factors depends on table executions_aggro
	constraint executions_agg_aggro_id_61c93502bce6f35f_fk_executions_aggro_id on table executions_aggro_state depends on table executions_aggro
2018-08-14 17:32:11.619 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:11.619 EDT [16873] STATEMENT:  drop table "public"."executions_aggro"
2018-08-14 17:32:12.034 EDT [16873] ERROR:  cannot drop table executions_case because other objects depend on it
2018-08-14 17:32:12.034 EDT [16873] DETAIL:  constraint executions_proof_case_id_65a8a882800b90d4_fk_executions_case_id on table executions_proofed_case_summary depends on table executions_case
	constraint executions_case__case_id_7cb8dc63a8d73a91_fk_executions_case_id on table executions_case_codefendants depends on table executions_case
	constraint executions_case__case_id_57f42b98c906a784_fk_executions_case_id on table executions_case_defense depends on table executions_case
	constraint executions_case_e_case_id_270743e0d6f0510_fk_executions_case_id on table executions_case_events depends on table executions_case
	constraint executions_case__case_id_71915cd90bb95c6d_fk_executions_case_id on table executions_case_links depends on table executions_case
	constraint executions_case__case_id_5562aba18394ace9_fk_executions_case_id on table executions_case_prosecutor depends on table executions_case
	constraint executions_case__case_id_1ba9e356be7f7bd7_fk_executions_case_id on table executions_case_victim depends on table executions_case
	constraint executions_case__case_id_2cca27f487c5fe3f_fk_executions_case_id on table executions_case_judge depends on table executions_case
	constraint executions_case__case_id_579704ed63ec9436_fk_executions_case_id on table executions_case_da depends on table executions_case
	constraint executions_pubbe_case_id_2b448f176cbb3981_fk_executions_case_id on table executions_pubbed_tweet depends on table executions_case
	constraint executions_sched__case_id_a842551a3644199_fk_executions_case_id on table executions_sched_tweet depends on table executions_case
	constraint executions_case__case_id_17078a846b2e33d0_fk_executions_case_id on table executions_case_bylines depends on table executions_case
	constraint executions_case__case_id_4d3a12adc65fdea0_fk_executions_case_id on table executions_case_factors depends on table executions_case
	constraint executions_share_case_id_478a3012cee2066b_fk_executions_case_id on table executions_shareable depends on table executions_case
2018-08-14 17:32:12.034 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:12.034 EDT [16873] STATEMENT:  drop table "public"."executions_case"
2018-08-14 17:32:12.700 EDT [16873] ERROR:  cannot drop table executions_city because other objects depend on it
2018-08-14 17:32:12.700 EDT [16873] DETAIL:  constraint executions__city_name_id_762afdc074a0b7fe_fk_executions_city_id on table executions_facility depends on table executions_city
	constraint executions_p_hometown_id_27c59bdc91db92e5_fk_executions_city_id on table executions_person depends on table executions_city
	constraint executions_agenc_city_id_361f3324c1e39fbf_fk_executions_city_id on table executions_agency depends on table executions_city
2018-08-14 17:32:12.700 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:12.700 EDT [16873] STATEMENT:  drop table "public"."executions_city"
2018-08-14 17:32:13.112 EDT [16873] ERROR:  cannot drop table executions_county because other objects depend on it
2018-08-14 17:32:13.112 EDT [16873] DETAIL:  constraint executions_c_county_id_4407cf5946df7c68_fk_executions_county_id on table executions_city depends on table executions_county
	constraint executions_c_county_id_3057003b03bc43ff_fk_executions_county_id on table executions_case depends on table executions_county
2018-08-14 17:32:13.112 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:13.112 EDT [16873] STATEMENT:  drop table "public"."executions_county"
2018-08-14 17:32:13.529 EDT [16873] ERROR:  cannot drop table executions_knelluser because other objects depend on it
2018-08-14 17:32:13.529 EDT [16873] DETAIL:  constraint execut_knelluser_id_1cdd4d8665d992f4_fk_executions_knelluser_id on table executions_knelluser_employer depends on table executions_knelluser
	constraint execut_knelluser_id_49f33138d8fb8138_fk_executions_knelluser_id on table executions_state_status_bylines depends on table executions_knelluser
	constraint execut_knelluser_id_2b5cae93db2f77cf_fk_executions_knelluser_id on table executions_proofed_case_summary_bylines depends on table executions_knelluser
	constraint execut_knelluser_id_59a625c9bf902d30_fk_executions_knelluser_id on table executions_proofed_state_status_bylines depends on table executions_knelluser
2018-08-14 17:32:13.529 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:13.529 EDT [16873] STATEMENT:  drop table "public"."executions_knelluser"
2018-08-14 17:32:13.956 EDT [16873] ERROR:  cannot drop table executions_newsorg because other objects depend on it
2018-08-14 17:32:13.956 EDT [16873] DETAIL:  constraint executions_newsorg_id_3da622e3bb037af5_fk_executions_newsorg_id on table executions_newsorg_state depends on table executions_newsorg
2018-08-14 17:32:13.956 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:13.956 EDT [16873] STATEMENT:  drop table "public"."executions_newsorg"
2018-08-14 17:32:14.374 EDT [16873] ERROR:  cannot drop table executions_person because other objects depend on it
2018-08-14 17:32:14.374 EDT [16873] DETAIL:  constraint executions_defendant_id_7816676b59aeb81_fk_executions_person_id on table executions_case depends on table executions_person
2018-08-14 17:32:14.374 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:14.374 EDT [16873] STATEMENT:  drop table "public"."executions_person"
2018-08-14 17:32:14.765 EDT [16873] ERROR:  cannot drop table executions_photo because other objects depend on it
2018-08-14 17:32:14.765 EDT [16873] DETAIL:  constraint executions_p_mugshot_id_2a877c4db3b5d5ee_fk_executions_photo_id on table executions_person depends on table executions_photo
	constraint executions_news_logo_id_653d1499a61ab5c0_fk_executions_photo_id on table executions_newsorg depends on table executions_photo
2018-08-14 17:32:14.765 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:14.765 EDT [16873] STATEMENT:  drop table "public"."executions_photo"
2018-08-14 17:32:15.173 EDT [16873] ERROR:  cannot drop table executions_proofed_case_summary because other objects depend on it
2018-08-14 17:32:15.173 EDT [16873] DETAIL:  constraint f3dd3acd12a32cd2a10b247d6c53832e on table executions_proofed_case_summary_bylines depends on table executions_proofed_case_summary
2018-08-14 17:32:15.173 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:15.173 EDT [16873] STATEMENT:  drop table "public"."executions_proofed_case_summary"
2018-08-14 17:32:15.611 EDT [16873] ERROR:  cannot drop table executions_proofed_state_status because other objects depend on it
2018-08-14 17:32:15.611 EDT [16873] DETAIL:  constraint ae2f649ba04c5b54f6994354d0f9bdeb on table executions_proofed_state_status_bylines depends on table executions_proofed_state_status
2018-08-14 17:32:15.611 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:15.611 EDT [16873] STATEMENT:  drop table "public"."executions_proofed_state_status"
2018-08-14 17:32:16.052 EDT [16873] ERROR:  cannot drop table executions_state because other objects depend on it
2018-08-14 17:32:16.052 EDT [16873] DETAIL:  constraint executions_sta_state_id_29178afa00d46c91_fk_executions_state_id on table executions_state_status depends on table executions_state
	constraint executions_pro_state_id_522c3b401cfad037_fk_executions_state_id on table executions_proofed_state_status depends on table executions_state
	constraint executions_per_state_id_4dc68aafecbfa522_fk_executions_state_id on table executions_person depends on table executions_state
	constraint executions_cou_state_id_4c20b864e5bbb55f_fk_executions_state_id on table executions_county depends on table executions_state
	constraint execution_state_name_id_3806211792ac855d_fk_executions_state_id on table executions_agency depends on table executions_state
2018-08-14 17:32:16.052 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:16.052 EDT [16873] STATEMENT:  drop table "public"."executions_state"
2018-08-14 17:32:16.539 EDT [16873] ERROR:  cannot drop table executions_state_status because other objects depend on it
2018-08-14 17:32:16.539 EDT [16873] DETAIL:  constraint e_state_status_id_3f47fc3702756e0_fk_executions_state_status_id on table executions_state_status_bylines depends on table executions_state_status
2018-08-14 17:32:16.539 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:16.539 EDT [16873] STATEMENT:  drop table "public"."executions_state_status"
2018-08-14 17:32:30.548 EDT [16873] ERROR:  cannot drop table executions_state because other objects depend on it
2018-08-14 17:32:30.548 EDT [16873] DETAIL:  constraint executions_per_state_id_4dc68aafecbfa522_fk_executions_state_id on table executions_person depends on table executions_state
	constraint executions_cou_state_id_4c20b864e5bbb55f_fk_executions_state_id on table executions_county depends on table executions_state
	constraint execution_state_name_id_3806211792ac855d_fk_executions_state_id on table executions_agency depends on table executions_state
2018-08-14 17:32:30.548 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:30.548 EDT [16873] STATEMENT:  drop table "public"."executions_state"
2018-08-14 17:32:36.759 EDT [16873] ERROR:  cannot drop table auth_user because other objects depend on it
2018-08-14 17:32:36.759 EDT [16873] DETAIL:  constraint executions_knelluser_user_id_7948349a51d96be2_fk_auth_user_id on table executions_knelluser depends on table auth_user
2018-08-14 17:32:36.759 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:36.759 EDT [16873] STATEMENT:  drop table "public"."auth_user"
2018-08-14 17:32:44.698 EDT [16873] ERROR:  cannot drop table auth_user because other objects depend on it
2018-08-14 17:32:44.698 EDT [16873] DETAIL:  constraint executions_knelluser_user_id_7948349a51d96be2_fk_auth_user_id on table executions_knelluser depends on table auth_user
2018-08-14 17:32:44.698 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:32:44.698 EDT [16873] STATEMENT:  drop table "public"."auth_user"
2018-08-14 17:33:05.364 EDT [16873] ERROR:  cannot drop table executions_photo because other objects depend on it
2018-08-14 17:33:05.364 EDT [16873] DETAIL:  constraint executions_p_mugshot_id_2a877c4db3b5d5ee_fk_executions_photo_id on table executions_person depends on table executions_photo
2018-08-14 17:33:05.364 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:33:05.364 EDT [16873] STATEMENT:  drop table "public"."executions_photo"
2018-08-14 17:33:08.909 EDT [16873] ERROR:  cannot drop table executions_city because other objects depend on it
2018-08-14 17:33:08.909 EDT [16873] DETAIL:  constraint executions_p_hometown_id_27c59bdc91db92e5_fk_executions_city_id on table executions_person depends on table executions_city
2018-08-14 17:33:08.909 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:33:08.909 EDT [16873] STATEMENT:  drop table "public"."executions_city"
2018-08-14 17:33:15.116 EDT [16873] ERROR:  cannot drop table django_celery_beat_crontabschedule because other objects depend on it
2018-08-14 17:33:15.116 EDT [16873] DETAIL:  constraint dj_crontab_id_d3cba168_fk_django_celery_beat_crontabschedule_id on table django_celery_beat_periodictask depends on table django_celery_beat_crontabschedule
2018-08-14 17:33:15.116 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:33:15.116 EDT [16873] STATEMENT:  drop table "public"."django_celery_beat_crontabschedule"
2018-08-14 17:33:15.905 EDT [16873] ERROR:  cannot drop table django_celery_beat_intervalschedule because other objects depend on it
2018-08-14 17:33:15.905 EDT [16873] DETAIL:  constraint interval_id_a8ca27da_fk_django_celery_beat_intervalschedule_id on table django_celery_beat_periodictask depends on table django_celery_beat_intervalschedule
2018-08-14 17:33:15.905 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:33:15.905 EDT [16873] STATEMENT:  drop table "public"."django_celery_beat_intervalschedule"
2018-08-14 17:33:21.965 EDT [16873] ERROR:  cannot drop table executions_city because other objects depend on it
2018-08-14 17:33:21.965 EDT [16873] DETAIL:  constraint executions_p_hometown_id_27c59bdc91db92e5_fk_executions_city_id on table executions_person depends on table executions_city
2018-08-14 17:33:21.965 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:33:21.965 EDT [16873] STATEMENT:  drop table "public"."executions_city"
2018-08-14 17:33:24.765 EDT [16873] ERROR:  cannot drop table executions_state because other objects depend on it
2018-08-14 17:33:24.765 EDT [16873] DETAIL:  constraint executions_per_state_id_4dc68aafecbfa522_fk_executions_state_id on table executions_person depends on table executions_state
	constraint executions_cou_state_id_4c20b864e5bbb55f_fk_executions_state_id on table executions_county depends on table executions_state
2018-08-14 17:33:24.765 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:33:24.765 EDT [16873] STATEMENT:  drop table "public"."executions_state"
2018-08-14 17:33:27.561 EDT [16873] ERROR:  cannot drop table executions_photo because other objects depend on it
2018-08-14 17:33:27.561 EDT [16873] DETAIL:  constraint executions_p_mugshot_id_2a877c4db3b5d5ee_fk_executions_photo_id on table executions_person depends on table executions_photo
2018-08-14 17:33:27.561 EDT [16873] HINT:  Use DROP ... CASCADE to drop the dependent objects too.
2018-08-14 17:33:27.561 EDT [16873] STATEMENT:  drop table "public"."executions_photo"
2018-08-14 17:34:09.781 EDT [24767] ERROR:  update or delete on table "django_content_type" violates foreign key constraint "auth_permiss_content_type_id_2f476e4b_fk_django_content_type_id" on table "auth_permission"
2018-08-14 17:34:09.781 EDT [24767] DETAIL:  Key (id)=(36) is still referenced from table "auth_permission".
2018-08-14 17:34:09.781 EDT [24767] STATEMENT:  delete from "public"."django_content_type" where "id"='36'
2018-08-18 15:58:24.975 EDT [18511] ERROR:  column crime_release.quarter does not exist at character 307
2018-08-18 15:58:24.975 EDT [18511] STATEMENT:  SELECT "crime_release"."id", "crime_release"."file_name", "crime_release"."file_type", "crime_release"."hj_url", "crime_release"."length", "crime_release"."date_released", "crime_release"."date_collected", "crime_release"."year_of_data", "crime_release"."frequency_type", "crime_release"."data_source_id", "crime_release"."quarter" FROM "crime_release" WHERE "crime_release"."frequency_type" = '2'
2018-08-18 16:52:42.786 EDT [26376] ERROR:  duplicate key value violates unique constraint "django_content_type_app_label_model_76bd3d3b_uniq"
2018-08-18 16:52:42.786 EDT [26376] DETAIL:  Key (app_label, model)=(crime, taskhistory) already exists.
2018-08-18 16:52:42.786 EDT [26376] STATEMENT:  UPDATE "django_content_type" SET "app_label" = 'crime', "model" = 'taskhistory' WHERE "django_content_type"."id" = 14
2018-08-18 16:52:42.787 EDT [26376] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-18 16:52:42.787 EDT [26376] STATEMENT:  SELECT 1
2018-08-18 16:54:24.282 EDT [26419] ERROR:  update or delete on table "django_content_type" violates foreign key constraint "auth_permission_content_type_id_2f476e4b_fk_django_co" on table "auth_permission"
2018-08-18 16:54:24.282 EDT [26419] DETAIL:  Key (id)=(19) is still referenced from table "auth_permission".
2018-08-18 16:54:24.282 EDT [26419] STATEMENT:  delete from "public"."django_content_type" where "id"='19'
2018-08-18 16:55:02.309 EDT [26596] ERROR:  insert or update on table "django_flatpage_sites" violates foreign key constraint "django_flatpage_sites_site_id_bfd8ea84_fk_django_site_id"
2018-08-18 16:55:02.309 EDT [26596] DETAIL:  Key (site_id)=(2) is not present in table "django_site".
2018-08-18 16:55:02.309 EDT [26596] STATEMENT:  SET CONSTRAINTS ALL IMMEDIATE
2018-08-18 16:55:02.310 EDT [26596] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-18 16:55:02.310 EDT [26596] STATEMENT:  SELECT 1
2018-08-18 16:55:39.474 EDT [26708] ERROR:  insert or update on table "django_flatpage_sites" violates foreign key constraint "django_flatpage_sites_site_id_bfd8ea84_fk_django_site_id"
2018-08-18 16:55:39.474 EDT [26708] DETAIL:  Key (site_id)=(1) is not present in table "django_site".
2018-08-18 16:55:39.474 EDT [26708] STATEMENT:  SET CONSTRAINTS ALL IMMEDIATE
2018-08-18 16:55:39.474 EDT [26708] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-18 16:55:39.474 EDT [26708] STATEMENT:  SELECT 1
2018-08-18 16:58:08.748 EDT [26991] ERROR:  insert or update on table "django_admin_log" violates foreign key constraint "django_admin_log_user_id_c564eba6_fk_auth_user_id"
2018-08-18 16:58:08.748 EDT [26991] DETAIL:  Key (user_id)=(1) is not present in table "auth_user".
2018-08-18 16:58:08.748 EDT [26991] STATEMENT:  SET CONSTRAINTS ALL IMMEDIATE
2018-08-18 16:58:08.748 EDT [26991] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2018-08-18 16:58:08.748 EDT [26991] STATEMENT:  SELECT 1
2018-08-20 08:47:15.025 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:15.025 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = '799d70f1-0261-4a0c-994a-2ebcaa1ce33a'
2018-08-20 08:47:15.029 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:15.029 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = '799d70f1-0261-4a0c-994a-2ebcaa1ce33a'
2018-08-20 08:47:15.029 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:15.029 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = '799d70f1-0261-4a0c-994a-2ebcaa1ce33a'
2018-08-20 08:47:15.056 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:15.056 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = '50a33b79-ebaa-4523-857e-acc1fb1ca66e'
2018-08-20 08:47:15.058 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:15.058 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = '50a33b79-ebaa-4523-857e-acc1fb1ca66e'
2018-08-20 08:47:15.059 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:15.059 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = '50a33b79-ebaa-4523-857e-acc1fb1ca66e'
2018-08-20 08:47:15.065 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:15.065 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = 'aaa20270-54ea-40e6-8152-d8ea0bd45939'
2018-08-20 08:47:15.066 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:15.066 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = 'aaa20270-54ea-40e6-8152-d8ea0bd45939'
2018-08-20 08:47:15.067 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:15.067 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = 'aaa20270-54ea-40e6-8152-d8ea0bd45939'
2018-08-20 08:47:15.073 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:15.073 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = '61f8ba1f-a4dd-4380-ad53-ec45ebae61a0'
2018-08-20 08:47:15.074 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:15.074 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = '61f8ba1f-a4dd-4380-ad53-ec45ebae61a0'
2018-08-20 08:47:15.075 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:15.075 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = '61f8ba1f-a4dd-4380-ad53-ec45ebae61a0'
2018-08-20 08:47:16.088 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:16.088 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = 'dc83c631-f0b9-4370-a0d3-57096ff69e01'
2018-08-20 08:47:16.090 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:16.090 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = 'dc83c631-f0b9-4370-a0d3-57096ff69e01'
2018-08-20 08:47:16.091 EDT [40688] ERROR:  relation "django_celery_results_taskresult" does not exist at character 479
2018-08-20 08:47:16.091 EDT [40688] STATEMENT:  SELECT "django_celery_results_taskresult"."id", "django_celery_results_taskresult"."task_id", "django_celery_results_taskresult"."status", "django_celery_results_taskresult"."content_type", "django_celery_results_taskresult"."content_encoding", "django_celery_results_taskresult"."result", "django_celery_results_taskresult"."date_done", "django_celery_results_taskresult"."traceback", "django_celery_results_taskresult"."hidden", "django_celery_results_taskresult"."meta" FROM "django_celery_results_taskresult" WHERE "django_celery_results_taskresult"."task_id" = 'dc83c631-f0b9-4370-a0d3-57096ff69e01'
2018-08-23 09:15:45.941 EDT [66504] ERROR:  relation "crime_scraped" does not exist at character 185
2018-08-23 09:15:45.941 EDT [66504] STATEMENT:  SELECT "crime_scraped"."id", "crime_scraped"."source_url_id", "crime_scraped"."html_hash", "crime_scraped"."s3_url", "crime_scraped"."scraped_date", "crime_scraped"."status_code" FROM "crime_scraped" ORDER BY "crime_scraped"."id" ASC LIMIT 1
2018-08-31 15:48:01.783 EDT [54018] LOG:  received smart shutdown request
2018-08-31 15:48:01.789 EDT [54018] LOG:  worker process: logical replication launcher (PID 54025) exited with exit code 1
2018-08-31 15:48:01.789 EDT [54020] LOG:  shutting down
2018-08-31 15:48:01.810 EDT [54018] LOG:  database system is shut down
2018-09-07 13:37:14.696 EDT [14599] LOG:  listening on IPv6 address "::1", port 5432
2018-09-07 13:37:14.697 EDT [14599] LOG:  listening on IPv6 address "fe80::1%lo0", port 5432
2018-09-07 13:37:14.697 EDT [14599] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2018-09-07 13:37:14.699 EDT [14599] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2018-09-07 13:37:14.755 EDT [14600] LOG:  database system was shut down at 2018-08-31 15:48:01 EDT
2018-09-07 13:37:14.763 EDT [14599] LOG:  database system is ready to accept connections
2019-03-15 21:20:46.525 EDT [14605] LOG:  stats_timestamp 2019-03-15 21:20:47.013694-04 is later than collector's time 2019-03-15 21:20:46.4696-04 for database 0
